docs/contracts/v4/guides/02-manage-liquidity/04-collect.mdx

```diff
@@ -1,3 +1,66 @@
 ---
 title: Collect Fees
----
\ No newline at end of file
+---
+
+### Setup
+
+See the [setup guide](./00-setup-liquidity.mdx)
+
+# Guide
+
+In order to collect fees, the integrator must execute encoded actions
+using the `PositionManager` contract.
+
+### 1. Import and define `IPositionManager`
+
+```solidity
+import {IPositionManager} from "v4-periphery/src/interfaces/IPositionManager.sol";
+// inside a contract, test, or foundry script:
+IPositionManager posm = IPositionManager(<address>);
+```
+
+### 2. Encode actions
+
+To collect fees, only the *decrease* operation is required.
+
+```solidity
+import {Actions} from "v4-periphery/src/libraries/Actions.sol";
+bytes memory actions = abi.encode(Actions.DECREASE_LIQUIDITY);
+```
```

Comment:
if you look in the Mint Position guide, we actually need to encode the token transfer. for fee collection its TAKE_PAIR (assuming the user is collecting both tokens)